projects/storefrontlib/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -229,7 +230,7 @@ export class SearchBoxComponent implements OnInit, OnDestroy {
   }
 
   // Focus on next item in results list
-  focusNextChild(event: UIEvent) {
+  focusNextChild(event: UIEvent | Event) {
```

Comment:
If the param needs to be type `Event` shouldn't we delete `UIEvent` from the signature since `UIEvent` extends `Event` anyways ?